lighthouse-core/audits/consistently-interactive.js

```diff
@@ -33,8 +33,9 @@ class ConsistentlyInteractiveMetric extends Audit {
       category: 'Performance',
       name: 'consistently-interactive',
       description: 'Consistently Interactive (beta)',
-      helpText: 'The point at which most network resources have finished loading and the ' +
-          'CPU is idle for a prolonged period.',
+      helpText: 'Consistently Interactive marks the time at which the page is ' +
+          'interactive enough to handle all user input. ' +
+          '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/consistently-interactive).',
```

Comment:
CI also depends on network idle. (specifically, we've reached just 2 remaining open network requests).

I don't think this detail needs to be here in the helptext, but it should probably be in the docs.

Furthermore, the docs say "Everything shown on the screen is interactive." but we're not testing for this really. We assume it, since main thread is idle.